control-plane-pki-operator/certificates/authority/policies.go

```diff
@@ -0,0 +1,185 @@
+/*
```

Comment:
Do we have a policy for wholesale copy-paste of modules? Upstream made it clear they would not be OK with exporting this.

Reply:
I would at least add a comment saying where this came from